src/HydrationRepo.js

```diff
@@ -36,16 +27,23 @@ class HydrationRepo {
     return totalOz / entries.length
   }
 
+  findWeeklyHydration(userID, endDate) {
+    const entries = this.findDataByID(userID)
+    const endingIndex = entries.map(singleEntry => {
+      return singleEntry.date
```

Comment:
The relocation and reorganization of this method is great! As are the rest of the changes you made, makes the file a LOT more readable.